### PR TITLE
Lsmallwood/search for webmap

### PR DESCRIPTION
**Pull request by lsmallwood**
@jtrieu please review and merge the first pass for this sample

### CODE REVIEW

File: ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#Search for webmap by keyword
+
+This sample demonstrates how find portal items by using a keyword, and limit the results to items of a certain type - in this case, webmaps.
+
+![](screenshot.png)
+
+##How it works
+1. A `Portal` is created up with a `Credential` using `OAuthModeUser`.
+2. When the app starts, the portal is loaded and the user is prompted to login.
+3. A search bar allows the user to enter a keyword (or tag) to search for - e.g. "usa". NOTE if multiple item types are required these can be set via the `types` property.
```
Comment (tdunn) [inline ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md]: Are keywords, tags, and item types all the same in this context? I think keywords and tags mean the same thing. If so, say `...enter a keyword (a tag) to ...`

File: ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+4. The search term is bound to the searchString for a `PortalQueryParametersForItems` which also has the property `type: Enums.PortalItemTypeWebMap`
+5. The query parameters are passed to the portal's `findItems` method.
+6. Once the findItems Task is complete, a `PortalQueryResultSetForItems` is obtained from the portal's `findItemsResult` property.
+7. The `PortalItemListModel` from the itemsResult is set on a QML ListView to display the list of webmaps.
```
Comment (tdunn) [inline ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md]: The official term is "web map" instead of "webmap". 
(Multiple places.)

Comment (tdunn) [inline ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md]: Also, it should be "web map" in the images.

File: ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+![](screenshot.png)
+
+##How it works
+1. A `Portal` is created up with a `Credential` using `OAuthModeUser`.
+2. When the app starts, the portal is loaded and the user is prompted to login.
+3. A search bar allows the user to enter a keyword (or tag) to search for - e.g. "usa". NOTE if multiple item types are required these can be set via the `types` property.
+4. The search term is bound to the searchString for a `PortalQueryParametersForItems` which also has the property `type: Enums.PortalItemTypeWebMap`
+5. The query parameters are passed to the portal's `findItems` method.
+6. Once the findItems Task is complete, a `PortalQueryResultSetForItems` is obtained from the portal's `findItemsResult` property.
```
Comment (tdunn) [inline ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md]: "When the `findItems` task is complete..."

Make the use of backward apostrophes for class names and members consistent throughout, please.

**Comment by jtrieu** [comment]
@lsmallwood it looks good to me. @tdunn if it looks good to you. can you merge?

**Comment by tdunn** [comment]
@lsmallwood - PR looks good. Thanks for doing this.

**Pull request merged**